sqldelight-compiler/src/test/kotlin/app/cash/sqldelight/core/AsyncTest.kt

```diff
@@ -0,0 +1,214 @@
+package app.cash.sqldelight.core
+
+import app.cash.sqldelight.test.util.FixtureCompiler
+import com.google.common.truth.Truth
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import java.io.File
+
+class AsyncTest {
```

Comment:
I think we can probably drop this test, I trust that the other ones which cover selects/inserts/updates/etc will provide enough coverage